htdocs/core/class/commondocgenerator.class.php

```diff
@@ -98,7 +99,7 @@ abstract class CommonDocGenerator
 	public $marge_droite;
 	public $marge_haute;
 	public $marge_basse;
-
+	public $corner_radius;
```

Comment:
@sonikf can you add phpdoc for the property ?

Reply:
Done